generators/_init-hybrid/templates/_gitignore

```diff
@@ -14,6 +13,9 @@ __pycache__/
 *.pyc
 *.swp
 *.lock
+/.awcache
+/local_cache/*
+/data/*
```

Comment:
```suggestion
```

We cannot ignore the data directory of a plugin, since this feature is used, e.g., in [taco_server](https://github.com/Caleydo/taco_server/tree/master/data) and [gapminder](https://github.com/Caleydo/gapminder/tree/master/data) and when building the product:

https://github.com/phovea/generator-phovea/blob/3a25093aec527c9edb5a89472cbb09f84dfc3408/generators/init-product/templates/plain/build.js#L175